packages/components/src/components/nav-item/nav-item.styles.ts

```diff
@@ -158,6 +158,13 @@ export default css`
     rotate: -180deg;
   }
 
+  /**
+   * Links should always show a chevron pointing to the right
+   */
+  .nav-item--is-link .nav-item__chevron {
+    rotate: -90deg;
+  }
```

Comment:
Don't we only want the chevron to show downwards / upwards in case of its an accordion? Right now, if not having a link and using the chevron attribute it would show upwards / downwards depending of the open attribute state.
![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/ec400dc7-1a9a-4928-8544-e1b659031932)

For example if someone is using the Angular `routerLink` i would assume the chevron wpuld also show to the right.

Reply:
Fixed with latest push. Please verify :)